src/Pages/ChannelRequests.razor

```diff
@@ -320,13 +321,13 @@
 
     // New Request integration
     private List<Wallet> _allWallets = new List<Wallet>();
-    protected DataGrid<ChannelOperationRequest> datagridRef;
+    protected DataGrid<ChannelOperationRequest>? datagridRef;
     private List<Node>? _manageableNodes;
     private int _selectedSourceNodeId;
     private Node? _selectedDestNode;
     private int _selectedWalletId;
     private string? _destNodeName;
-    private long _amount = 20000L;
```

Comment:
Just out of curiosity, why did you remove this?

Reply:
Because to reset the modal I would have to again set the _amount variable to 20000L. 
If I duplicate this assignment, next time someone has to change it, it has to do it in 2 or more places.
Now I only assign it in the `ResetChannelRequestRejectModal` method, and then call that method on `OnInitializedAsync` and everywhere else that I needed.

Another option (maybe better) would be to create a "Constants" file, in which we store all this "magic numbers"

Reply:
I think this might be the minimum channel capacity that is set as an env var. Maybe we could use that. `MINIMUM_CHANNEL_CAPACITY_SATS: 20000` is set in the docker compose file.

Reply:
awesome, I'll use that